src/main/java/org/primefaces/expression/SearchExpressionConstants.java

```diff
@@ -33,7 +35,7 @@
     public static final String ROW_KEYWORD =                KEYWORD_PREFIX + "row";
     public static final String COLUMN_KEYWORD =             KEYWORD_PREFIX + "column";
     public static final String ID_KEYWORD =                 KEYWORD_PREFIX + "id";
-    public static final String ROOT_KEYWORD =               KEYWORD_PREFIX + "root";
+    public static final String ROOT_KEYWORD =               KEYWORD_PREFIX + ROOT_ROW_KEY;

```

Comment:
You can remove this one as SEF keyword root may change independently of the Tree root rowkey.

Reply:
@melloware My bad, removed and updated PR